package.json

```diff
@@ -35,6 +35,7 @@
     "react": "0.12.2",
     "serve-static": "1.9.2",
     "shelljs": "0.4.0",
+    "sinon": ">=1.4.0 <2",
```

Comment:
Okay, then I'll use the specific version here.  As far as why sinon is used, it is a peer dependency of sinon-chai.  I have no idea why simon-chai is a dependency.